pkg/driver/nri_hooks.go

```diff
@@ -168,11 +168,6 @@ func (cp *CPUDriver) CreateContainer(_ context.Context, pod *api.PodSandbox, ctr
 		setSharedCPUUpdateRequired(false)
 	}
 
-	if isSharedCPUUpdateRequired() {
-		updates = cp.getSharedContainerUpdates(containerId)
-		setSharedCPUUpdateRequired(false)
-	}
```

Comment:
this seems correct. I' confident a definitive fix is coming with https://github.com/kubernetes-sigs/dra-driver-cpu/pull/26 which I'll resume working ASAP. Mostly misses testing, but now we have the infra for them.